VirtualReality/Widgets/qMRMLVirtualRealityHomeWidget.h

```diff
@@ -66,16 +70,24 @@ public slots:
   //void updateViewFromReferenceViewCamera();
   //void setMagnificationLock(bool);
 
+  void setupModuleButton(QWidget* widget, QIcon* icon);
+  void onModuleButtonPressed();
+  void onBackButtonPressed();
+
 protected slots:
   /// Update widgets from the MRML node
   void updateWidgetFromMRML();
 
+
 protected:
   QScopedPointer<qMRMLVirtualRealityHomeWidgetPrivate> d_ptr;
+  
 
 private:
   Q_DECLARE_PRIVATE(qMRMLVirtualRealityHomeWidget);
   Q_DISABLE_COPY(qMRMLVirtualRealityHomeWidget);
+
+  QMap<QPushButton*, QWidget*> ModuleWidgetsMap;
```

Comment:
I think this should go into the private class. Remember, anything that the users of the public class shou;dn't have access to should be in the private class (the map is private anyway, and the idea of the private class is to minimize changes in the header file). If we need access to the map, we should add functions for that later.